addons/mail/static/src/new/composer/composer.js

```diff
@@ -340,6 +340,11 @@ export class Composer extends Component {
             this.attachmentUploader?.clear();
             this.props.composer.textInputContent = "";
             el.focus();
+        } else {
+            this.env.services.notification.add(
+                this.env._t("Please wait while the file is uploading."),
+                { type: "warning" }
```

Comment:
The issue is that this message will also be displayed when pressing `Enter` on an empty composer. I think this should be an `else if` instead.